thoth/storages/graph/postgres.py

```diff
@@ -3632,6 +3632,169 @@ def create_github_app_installation(self, slug: str, repo_name: str, private: boo
                 )
                 return newly_added
 
+    def get_kebechet_github_app_installations_all(
+        self,
+        id: Optional[int] = None,
```

Comment:
```suggestion
        *,
        id: Optional[int] = None,
```

Please make them required as keywords arguments.